src/mame/korg/korgtriton.cpp

```diff
@@ -0,0 +1,210 @@
+
+        // System DRAM
+        map(0x01000000, 0x01ffffff).ram().share(m_ram);
+    }
+
+    void korgtriton_state::korgtriton(machine_config &config)
+    {
+        SH7043(config, m_maincpu, 7_MHz_XTAL * 4);
+        m_maincpu->set_addrmap(AS_PROGRAM, &korgtriton_state::map);
+
+        m_maincpu->read_porta().set(FUNC(korgtriton_state::pa_r));
+        m_maincpu->write_porta().set(FUNC(korgtriton_state::pa_w));
+        m_maincpu->read_porte().set(FUNC(korgtriton_state::pe_r));
+        m_maincpu->write_porte().set(FUNC(korgtriton_state::pe_w));
+
+        // 320x240 screen, should be a simple framebuffer at LCDCIO
+        SCREEN(config, m_screen, SCREEN_TYPE_RASTER);
+        m_screen->set_refresh_hz(15);
+        m_screen->set_vblank_time(ATTOSECONDS_IN_USEC(0));
+        m_screen->set_size(320, 240);
+        m_screen->set_visarea(0, 319, 0, 239);
+        m_screen->set_screen_update(FUNC(korgtriton_state::screen_update));
+
+        I8251(config, m_scu, 1021800);
+        m_scu->txrdy_handler().set(FUNC(korgtriton_state::scu_txrdy_w));
+        m_scu->rxrdy_handler().set(FUNC(korgtriton_state::scu_rxrdy_w));
+    }
+
+    u32 korgtriton_state::pa_r()
+    {
+        return 0;
+    }
+
+    void korgtriton_state::pa_w(u32 data)
+    {
+        printf("pa_w: %08x\n", data);
+    }
+
+    u32 korgtriton_state::pe_r()
+    {
+        return 0;
+    }
+
+    void korgtriton_state::pe_w(u32 data)
+    {
+        printf("pe_w: %08x\n", data);
+    }
+
+    uint32_t korgtriton_state::screen_update(screen_device &screen, bitmap_rgb32 &bitmap, const rectangle &cliprect)
+    {
+        for (int y = cliprect.min_y; y <= cliprect.max_y; ++y)
+        {
+            uint32_t *scan = &bitmap.pix(y, 0);
+            for (int x = 0; x < 320; ++x)
+            scan[x] = m_lcdcio[y * 320 + x];
+        }
+        return 0;
+    }
+
+    void korgtriton_state::scu_txrdy_w(int state)
+    {
+        printf("upd71051_txrdy_w: %d\n", state);
+    }
+
+    void korgtriton_state::scu_rxrdy_w(int state)
+    {
+        printf("upd71051_rxrdy_w: %d\n", state);
+    }
+
+    // the following rom images are taken from the Triton OS 2.0.0 floppy disks
+    // the romfiles are missing the first two bytes (checksum) present in the
+    // floppy disks obtainable.
+
+    ROM_START( korgtriton )
+    ROM_REGION( 0x00800000, "maincpu", 0 )
+
+    ROM_LOAD("int01080.710", 0x00000000, 0x00008000, CRC(ef0d7b3a) SHA1(5b20814dbd118df746437a5aa0b9bc371b8bada0))
+    ROM_LOAD("int13080.710", 0x00008000, 0x00018000, CRC(295bd0f8) SHA1(5f21cbaa95d9622aa0cfcf8b453d8c98b5dc54cd))
+
+    ROM_LOAD("ext03080.710", 0x00400000, 0x000c0000, CRC(12e681b1) SHA1(fb3299cffd8fc11ccbf5d7228e1d35f8a824579c))
+    ROM_LOAD("ext34080.710", 0x004c0000, 0x00100000, CRC(a84ea53c) SHA1(c949bef347f0cf02a16a6a332aeedcff31c6b956))
+    ROM_LOAD("ext74080.710", 0x005c0000, 0x00100000, CRC(281cef3b) SHA1(f348fb53f2ce954fda8004d310566bec663374f4))
+    ROM_END
```

Comment:
Please indent the content of the ROM definitions by one level.  We treat `ROM_START`/`ROM_END` as delimiting a block.